core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -260,11 +260,11 @@ import scala.concurrent.duration._
       log.debug("Batch size reached. Writing to Cassandra.")
       write(buffer.take(settings.maxBatchSize), buffer.drop(settings.maxBatchSize), tagSequenceNrs, None)
     } else if (settings.flushInterval == Duration.Zero) {
-      log.debug("Flushing right away as interval is zero")
       // Should always be a buffer of 1
       write(buffer, Vector.empty[(Serialized, TagPidSequenceNr)], tagSequenceNrs, None)
     } else {
-      timers.startSingleTimer(FlushKey, InternalFlush, settings.flushInterval)
+      if (!timers.isTimerActive(FlushKey))
```

Comment:
Doh! Should we add a test to cover this?

Reply:
ok, I'll see what I can add

Reply:
added test 9dbf36f33b927879995fb5a226743a3eb11a1f28
(I hope it's not too timing sensitive)